Integrations/integration-ServiceNow.yml

```diff
@@ -2926,7 +2959,8 @@ script:
     - contextPath: Ticket.Number
       description: ServiceNow ticket number
       type: string
-    description: Retrieve ticket info with query
+    description: Deprecated. Please use servicenow-query-table or servicenow-query-tickets
```

Comment:
Please remove `please` :)

Reply:
@Itay4 I added that description to the old commands, this one specifically I forgot :)